ta/crypto_perf/include/ta_crypto_perf.h

```diff
@@ -0,0 +1,76 @@
+/* SPDX-License-Identifier: BSD-2-Clause */
+/*
+ * Copyright (c) 2023, Linaro Limited
+ * All rights reserved.
+ */
+
+#ifndef TA_CRYPTO_PERF_H
+#define TA_CRYPTO_PERF_H
+
+#define TA_CRYPTO_PERF_UUID { 0xe626662e, 0xc0e2, 0x485c, \
+	{ 0xb8, 0xc8, 0x09, 0xfb, 0xce, 0x6e, 0xdf, 0x3d } }
+
+/*
+ * Commands implemented by the TA
+ */
+
+#define TA_CIPHER_PERF_CMD_PREPARE_KEY			0
+#define TA_CIPHER_PERF_CMD_PROCESS			1
+#define TA_CIPHER_PERF_CMD_PROCESS_SDP			2
+#define TA_HASH_PERF_CMD_PREPARE_OP			3
+#define TA_HASH_PERF_CMD_PROCESS			4

```

Comment:
Please keep a common `TA_CRYPTO_PERF_CMD_` prefix, to be consistent with `TA_CRYPTO_PERF_UUID`.
```C
#define TA_CRYPTO_PERF_CMD_CIPHER_PREPARE_KEY	0
#define TA_CRYPTO_PERF_CMD_CIPHER_PROCESS	1
#define TA_CRYPTO_PERF_CMD_CIPHER_PROCESS_SDP	2
#define TA_CRYPTO_PERF_CMD_HASH_PREPARE_OP	3
#define TA_CRYPTO_PERF_CMD_HASH_PROCESS		4
...